commitlint.config.js

```diff
@@ -1 +1 @@
-module.exports = {extends: ['@commitlint/config-conventional']};
+module.exports = { extends: ['@commitlint/config-conventional'] };
```

Comment:
Is this [object-curly-spacing](https://eslint.org/docs/rules/object-curly-spacing) rule ?
If yes, should it been included in our .eslintrc rules ?

Reply:
I find the object-curly-space rule been configured as always in eslint-config-keyston .eslintrc file.
Thus I'll resolve this conversation directly.